app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntrySheetController.kt

```diff
@@ -493,6 +495,7 @@ class BloodPressureEntrySheetController @Inject constructor(
               .flatMap { bpSaved ->
                 appointmentRepository
                     .markAppointmentsCreatedBeforeTodayAsVisited(newBp.patientUuid)
+                    .andThen(patientRepository.updatePatientRecordedAt(newBp.patientUuid, toUtcInstant(newBp.date)))
```

Comment:
Instead of computing the instant again, why don't we add the saved `BloodPressureMeasurement` instance as a property of `BloodPressureSaved` and use the `recordedAt` from there?

Reply:
I would not want to do that since `BloodPressureSaved` is not for that purpose.
What we are doing with patient `recordedAt` is like a side effect as it does not affect the outcome of BP save in any way.